src/gstvimbasrc.c

```diff
@@ -1573,6 +1578,10 @@ VmbError_t set_roi(GstVimbaSrc *vimbasrc)
     }
 
     // offsety
+    if (vimbasrc->properties.offsety == INT_MAX) {
```

Comment:
Same as above for `offsetx`

```suggestion
    if (vimbasrc->properties.offsety == -1) {
```